src/ziggurat/init.clj

```diff
@@ -187,3 +191,6 @@
        (log/error e)
        (stop stop-fn modes)
        (System/exit 1)))))
+
+(defn -initJava [args]
```

Comment:
should we rename this to init? All other namespaces have the same function names.